x-pack/plugins/security_solution/public/management/components/administration_list_page.tsx

```diff
@@ -55,11 +65,18 @@ export const AdministrationListPage: FC<AdministrationListPageProps & CommonProp
           description={description}
           bottomBorder={true}
           rightSideItems={[actions]}
-          restrictWidth={false}
+          restrictWidth={restrictWidth}
```

Comment:
This seems strange to me. The restriction of the page width on this component (and as named) should be for the entire page - but its being applied ONLY to the page header. 

if that is the intent, then can we rename the prop to `restrictHeaderWidth`?

because it looks like the both of the page is always restricted (see below, line 78)

Reply:
good catch!  this should absolutely be used in the content as well, I'm pushing up a change to address this.